packages/reports/app/styles/navi-reports/components/filter-collection.less

```diff
@@ -146,4 +146,13 @@
       color: @navi-gray-600;
     }
   }
+
+  &--collapsed {
+    cursor: pointer;
+
+    &-item:not(:last-child):after {
+      content: ';';
```

Comment:
Maybe `.` would be a better separator. It fits better with the plain text style display of the collapsed filters. 
We could also add some whitespace between the last filter value and the next filter builder.